src/App.tsx

```diff
@@ -1,26 +1,44 @@
 import { FC, Suspense } from "react";
 import { Routes, Route } from "react-router-dom";
 
+import Loader from "./components/atoms/Loader";
 import Navbar from "./components/organisms/Navbar";
 import Footer from "./components/organisms/Footer";
 import ProblemsPage from "./components/pages/ProblemsPage";
 import ProblemDetailedPage from "./components/pages/ProblemDetailedPage";
 import AboutMethodPage from "./components/pages/AboutMethodPage";
 import Moderator from "./components/pages/Moderator";
+import LoginPage from "./components/pages/LoginPage";
+import { AuthProvider } from "./auth/AuthProvider";
+import RequireAuth from "./auth/RequireAuth";
 
 const App: FC = () => {
   return (
-    <Suspense fallback="loading">
-      <Navbar />
-      <Routes>
-        {/* <Route path="/" element {<LandingPage/>}> */}
-        <Route path="/" element={<ProblemsPage />} />
-        <Route path="/problems/:problemId" element={<ProblemDetailedPage />} />
-        <Route path="/about" element={<AboutMethodPage />} />
-        <Route path="/moderator" element={<Moderator />} />
-      </Routes>
-      <Footer />
-    </Suspense>
+    <AuthProvider>
+      <Suspense fallback={<Loader />}>
```

Comment:
Also works with <Suspense fallback={null}> I think ;)

Reply:
Could you please describe it more clearly? 🤔 I don't know what we should change here ;)